artemis-protocols/artemis-amqp-protocol/src/main/java/org/apache/activemq/artemis/protocol/amqp/broker/AMQPSessionCallback.java

```diff
@@ -134,6 +139,24 @@ public void run() {
       }
    }
 
+   public void withinContext(RunnableEx run) throws Exception {
```

Comment:
@jbertram RunnableEx is being used here through Lambdas...
(you asked me about that privately)